integration-tests/src/test/resources/test-project/model/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+/* * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import io.spine.internal.dependency.JUnit
+import io.spine.internal.dependency.Spine
+import io.spine.internal.dependency.Validation
+import io.spine.internal.dependency.HelloProtoData
+
+repositories {
+    mavenLocal()
+}
+
+dependencies {
+    // Enable field options extension.
+    api(HelloProtoData.extension)
+
+    // Add module with code generation plugin to ProtoData classpath.
+    protoData(HelloProtoData.plugin)
+
+    // To allow access to `ValidatingBuilder` from the generated Kotlin code.
+    implementation(Validation.runtime)
```

Comment:
The problem here is that we don't apply Validation as a ProtoData plugin. But for some reason, we still depend on `ValidatingBuilder`, and hence, on the `Validation.runtime`.

Let's discuss.